mono/metadata/class.c

```diff
@@ -995,7 +995,11 @@ inflate_generic_context (MonoGenericContext *context, MonoGenericContext *inflat
 MonoMethod *
 mono_class_inflate_generic_method (MonoMethod *method, MonoGenericContext *context)
 {
-	return mono_class_inflate_generic_method_full (method, NULL, context);
+	MonoError error;
+	MonoMethod *res = mono_class_inflate_generic_method_full_checked (method, NULL, context, &error);
+	if (!is_ok (&error))
+		g_error ("Could not inflate generic method due to %s", mono_error_get_message (&error));
```

Comment:
Can't we use `mono_error_assert_ok` here?

Reply:
Yes, but then we would only get the message from the MonoError, without the extra text; maybe I should make an analogue of `g_assertf` for `mono_error_assert_ok`.